### PR TITLE
Link to maintained successor for PHP

### PR DESCRIPTION
**Pull request by dereuromark**
Previous one is not maintained anymore.
So taking over as a group effort from now on.

### CODE REVIEW

**Comment by sindresorhus** [comment]
I think this is a bit soon. You need to first prove that you're a good maintainer for the list.

Also make sure the list passes the current guidelines and Awesome lint.

**Comment by dereuromark** [comment]
My track record of decades in open source should speak for itself :)
Also, as outlined, this should be a community thing moving forward, not a single bus factor Project.

**Comment by dereuromark** [comment]
Fun fact:
I am the creator and primary maintainer of https://github.com/FriendsOfCake/awesome-cakephp already - as I am a core member of CakePhp framework for a very long time already

**Review by sandeep09041999 — APPROVED** [review]

File: readme.md
Comment (shahardra) [inline readme.md:277]: $ node --experimental-permission t.js
node:internal/modules/cjs/loader:162
  const result = internalModuleStat(filename);
                 ^

Error: Access to this API has been restricted
    at stat (node:internal/modules/cjs/loader:162:18)
    at Module._findPath (node:internal/modules/cjs/loader:640:16)
    at resolveMainPath (node:internal/modules/run_main:15:25)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:53:24)
    at node:internal/main/run_main_module:23:47 {
  code: 'ERR_ACCESS_DENIED',
  permission: 'FileSystemRead',
  resource: '/Users/rafaelgss/repos/os/node/t.js'
}

**Review by shahardra — APPROVED** [review]

**Comment by dereuromark** [comment]
> Also make sure the list passes the current guidelines and Awesome lint.

I would usually agree with you.
But switching to a maintained list on an existing item (basically a 1:1 replacement, quite literally) should have a different flow than adding a completely new list.
As such I would hope we can fast track this one

Any further needs to get the linting green will of course be taken as a group effort in the very near future.
But status quo on the old list is the same, so I dont see the need to have this as a hard requirement at the time being.

**Comment by alexkart** [comment]
Is there a final decision on how this should be handled?

**Comment by ziadoz** [comment]
Hi. I'm the author of the original Awesome PHP list. 

It seems like @dereuromark and @alexkart tried to reach out in an [issue](https://github.com/ziadoz/awesome-php/issues/1231) on my repo, but I only saw it today. Sorry about that. I've had a busy and stressful 2023.

I would prefer bringing on new collaborators to my list over a hard fork. Will discuss further on the issue. Thanks.

**Comment by dereuromark** [comment]
Closing then for now since the original maintainer is back and we can collaborate.

**Pull request closed without merging**
